migrate/repo_checks.py

```diff
@@ -78,6 +97,253 @@ def dry_run(self):
         raise NotImplementedError
 
 
+class EnsureWorkflowTemplates(Check):
+    """
+    There are certain github action workflows that we to exist on all
+    repos exactly as they are defined in the `.github` repo in the org.
+
+    Check to see if they're in a repo and if not, make a pull request
+    to add them to the repository.
+    """
+
+    def __init__(self, api: GhApi, org: str, repo: str):
+        super().__init__(api, org, repo)
+
+        self.workflow_templates = [
+            "self-assign-issue.yml",
+            "add-depr-ticket-to-depr-board.yml",
+            "commitlint.yml",
+            "add-remove-label-on-comment.yml",
+        ]
```

Comment:
This check doesn't handle `$default-branch`. I don't think it'd be hard to fix, but if you'd rather not handle it right now, I think just a comment here would be A-OK.
```suggestion
        # Heads up: This repo check currently does not currently handle the $default-branch
        # placeholder. If you want to add a template here that uses $default-branch,
        # then you'll need to first make the repo check aware of $default-branch.
        self.workflow_templates = [
            "self-assign-issue.yml",
            "add-depr-ticket-to-depr-board.yml",
            "commitlint.yml",
            "add-remove-label-on-comment.yml",
        ]
```